docs/UserGuide.md

```diff
@@ -334,8 +334,19 @@ Examples:
 
 ![view](images/viewWarehouse.png)
 
+### Undo/Redo a previous editing : `undo`/`redo`
+
+Recovers the previous version before an editing on the CLI-nic data or restores the later version of edited data after
+a recovery to previous version.
+
+Format: `undo` / `redo`
+
+* Any trailing words behind the command words will be ignored.
+* For example: if one types `undo redo`, the command will be to `undo` the editing.
```

Comment:
Would this contradict with previous commands, where we accepts the last argument?

Reply:
Not really since the previous command accepts the last argument while this one accepts the first command word.

Reply:
I think it's alright here because it's easier to take just the first word which would be the command word, in the parser we can ignore everything that comes after since there are no params to process.

Reply:
okay then! 👍